### DISCUSSION TITLE
Very slow Running gatsby.IMAGE_CDN jobs

### DISCUSSION THREAD

**Discussion by rburgst** [gatsbyjs/gatsby#35327] [Help]
I have a relatively small wordpress site (however, it does have a LOT of images). And ever since upgrading to 4.10 (IIRC) I am seeing

```
Running gatsby.IMAGE_CDN jobs
```

and it takes like 15minutes on my M1 Pro Macbook Pro with 32GB RAM.

I tried setting 
```
export  GATSBY_CLOUD_IMAGE_CDN=0
```

but that didnt change anything.

Even on `gatsby-starter-wordpress-blog` shows up and takes a quite long time.

Here is the output (mid build) from my actual project:

```
[================            ]   980.241 s 11119/19224 58% Running gatsby.IMAGE_CDN jobs
[============================]   567.656 s 4146/4146 100% Running gatsby-plugin-sharp.IMAGE_PROCESSING jobs
```

After 15 min, only 58% complete?

What is it doing?

**Comment by rburgst** [comment]
In the end the build completed in 
```
✨  Done in 1464.32s.
```
The vast majority of the time was spent doing `IMAGE_CDN` jobs.

**Comment by LekoArts** [comment]
Please post your feedback in https://github.com/gatsbyjs/gatsby/discussions/35147
